crates/stackable-operator/src/commons/networking.rs

```diff
@@ -74,3 +166,48 @@ impl Deref for KerberosRealmName {
         &self.0
     }
 }
+
+#[cfg(test)]
+mod tests {
+    use super::*;
+
+    use rstest::rstest;
+
+    #[rstest]
+    #[case("foo")]
+    #[case("foo.bar")]
+    // This is also a valid domain name
+    #[case("1.2.3.4")]
```

Comment:
note: Actually, no. This is not a valid domain name. Domain name labels **must** start with one of the 52 alphabetic characters (A-Z and a-z).

Reply:
See https://www.rfc-editor.org/rfc/rfc1035#section-2.3.1